test/data/manifests/rhel_90-x86_64-qcow2-boot.json

```diff
@@ -10427,18 +10443,6 @@
       "missing": []
     },
     "selinux": {
-      "context-mismatch": [
-        {
-          "actual": "unconfined_u:object_r:etc_t:s0",
-          "expected": "system_u:object_r:etc_t:s0",
-          "filename": "/etc/sysconfig/kernel"
-        },
-        {
-          "actual": "unconfined_u:object_r:etc_t:s0",
-          "expected": "system_u:object_r:etc_t:s0",
-          "filename": "/etc/sysconfig/network"
-        }
-      ],
```

Comment:
How did you regenerate the image test case? I'm asking because the removal of this part looks suspicious to me. Especially because the SELinux stage in the RHEL-9.0 qcow2 pipeline is placed before the Sysconfig stage [1], which is incorrect and actually results in these two files having wrong context. Well, at least it should.

[1] https://github.com/osbuild/osbuild-composer/blob/b1e40b5ce7ead559adfcd8d9f6ee96fafbb6f503/internal/distro/rhel90/distro.go#L384-L397

Reply:
Weird, I just used the multi-arch generator you created, any ideas why this is wrong?

```
tools/test-case-generators/generate-all-test-cases \
  --output=test/data/manifests \
  --image-ppc64le ~/images/Fedora-Cloud-Base-33-1.2.ppc64le.qcow2 \
  --image-s390x ~/images/Fedora-Cloud-Base-33-1.2.s390x.qcow2 \
  --image-aarch64 ~/images/Fedora-Cloud-Base-33-1.2.aarch64.qcow2 \
  --image-x86_64 ~/images/Fedora-Cloud-Base-33-1.2.x86_64.qcow2 \
  --arch s390x ppc64le aarch64 x86_64 \
  --distro rhel-90 \
  --image-types qcow2
```

Reply:
The command looks good, but I have an idea of what might went wrong...

IIRC, F33 does not contain new enough SELinux tooling and it can't understand the policy version on RHEL-9. Only F34 or newer systems can. This was probably logged to the console by image-info, but I'm afraid that this went unnoticed when run in the VM using the script...

Reply:
Ahhhh, you are using F34 for this, right?

Reply:
Yes, exactly... :-/